package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "alchemy-client",
-  "version": "0.10.5",
+  "version": "0.10.6",
```

Comment:
a new version for exp 1.0.0

Reply:
1.0.0-exp-rc.1? Making it clear that this is still a release candidate and not official 1.0.0?